licenses/templates/macros_templates/engine.html

```diff
@@ -62,7 +62,7 @@
                               <div id="donation-bar-inner" class="donation-bar-inner">
                                 <p class="donate-text">Help us build a vibrant, collaborative global commons</p>
                                 <div class="donate-action">
-                                    <a href="https://us.netdonor.net/page/6650/donate/1?ea.tracking.id=chooser" class="button donate arrow">Donate<span class="hide-on-mobile"> Now</span></a>
+                                    <a href="https://creativecommons.org/donate?ea.tracking.id=chooser" class="button donate arrow">Donate<span class="hide-on-mobile"> Now</span></a>
```

Comment:
We aren't going to custom track donations from the legacy Chooser, so this should just be:
https://creativecommons.org/donate
You can remove the parameter, that's an Engaging Networks parameter that won't be passed through.